src/Uno.UI/UI/Xaml/Input/IHandleableRoutedEventArgs.cs

```diff
@@ -11,5 +11,7 @@ internal interface IHandleableRoutedEventArgs
 		/// route from handling the same event again.
 		/// </summary>
 		bool Handled { get; set; }
+
+		bool ShouldPreventDefaultIfHandled { get; set; }
```

Comment:
The advantage to expose an enum like the `HtmlEventDispatchResult` instead of a `bool` is that we are future-proof if we need later to add another distinction between "managed handled" and "native handled" ... but your approach is faster and easier and as this interface is internal so that good 👍

I would however revert the `bool` so we won't forget to set it by default to `true` for new event args:
```suggestion
		/// <summary>
		/// [WASM ONLY] 
		/// By default an event flagged as <see cref="Handled"/> will prevent the "default behavior" of the browser.
		/// Setting this flag to `true` will make sure that this "default behavior" will not be suppressed.
		/// cf. https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault
		/// </summary>
		bool DoNotPreventDefaultIfHandled { get; set; }
```